cwrapper/CArachneWrapperTest.cc

```diff
@@ -117,19 +117,27 @@ limitedTimeWait(std::function<bool()> condition, int numIterations) {
     ASSERT_TRUE(condition());
 }
 
+/**
+ * Functions below are used to test thread creation
+ */
 static void*
 wrapperMain(void* arg) {
-    *(unsigned*)arg = 0xDEADBEEF;
+    usleep(1000);
```

Comment:
Again, a comment explaining why this sleep exists would be helpful for the future.